### ISSUE TITLE
issues not detected but no parser error reported

### ISSUE THREAD

**Issue by schroeca** [SonarOpenCommunity/sonar-cxx#713]
i still have severe issues parsing my ***SystemC*** code and report all issues.

Example:

In one file I get the cxx:ReservedNames issue reported at following lines
```C++
#ifndef _TRX_SYSTEMC_PSP2I4O_PSP2I4O_V1_H_
#define _TRX_SYSTEMC_PSP2I4O_PSP2I4O_V1_H_
```

In another file i do not get this issue reported. The defines have the same structure.

```C++
#ifndef _TRX_SYSTEMC_SONARQUBE_TEST_SC_MODULE_TEST_H_
#define _TRX_SYSTEMC_SONARQUBE_TEST_SC_MODULE_TEST_H_

```

Example 2:

```C++
namespace trx_systemc {

SC_MODULE(test1) {

...

};
}

```
--> No issue reported

The second file looks exactly the same

```C++
namespace trx_systemc {

SC_MODULE(test_module2) {

...

};
}

```

--> Source code not correctly indented

Why?
Whats the recommend way of tracking down such issues of the sonar-cxx project??

**Comment by guwirth** [comment]
For reserved name check see: https://github.com/SonarOpenCommunity/sonar-cxx/blob/master/cxx-checks/src/main/java/org/sonar/cxx/checks/ReservedNamesCheck.java. The leading underscore seems to be the problem. 

> Whats the recommend way of tracking down such issues of the sonar-cxx project??

Get debug information: https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Get-debug-information

**Comment by schroeca** [comment]
It is not the leading underscore --> I do not understand why this issue is not detected in all cases. Sometimes it is an error and somtimes not. 

I tried to track the problem wihtin the debug log and I ended up with following finding.
The issue is within the log file but it is not displayed in sonarqube. I also checked if it was set to falsePositive, not the case!

In the debug-log this issue is surrounded by warnings that some resources can not be found.
Could this cause such a "problem"
